### ISSUE TITLE
Feature request: ability to download to csv

### ISSUE THREAD

**Issue by robertDurst** [Ameobea/robintrack#40]
Hey! So my friend is into stocks and wanted a way to pull some of this data into his spreadsheets so I wrote a small script that pulled data from y'alls site and outputed a nice csv:

https://github.com/robertDurst/popularityFromRobinTracker

Just wanted to say thanks for doing this and this is a nice feature to have if y'all have time.

Rock on :fist_raised:

**Comment by Ameobea** [comment]
Hey!  It's really cool that you built that tool; I'm really interested by the fact that your friend is using Robintrack data in his spreadsheets.  I hope he has success making it useful in his trading decisions.

For adding CSV exports to the site, I don't see any reason why not to do that.  It should be an easy addition.

In the future, I need to change the database to store all of the updates for a particular symbol in the same document rather than having thousands of documents that each contain a single snapshot.  That will make everything run way faster, but will be a very significant change.

Anyway, thanks again for taking the time to make that tool.  If you ever want a database dump or something, just get in touch and I can help you out.

**Comment by dalexj** [comment]
@Ameobea this one sounds interesting. let me know if you need any help with it

**Comment by robertDurst** [comment]
@Ameobea thanks for the quick response! 

Yeah it probably is a pretty easy addition and a useful one at that for those who only use excel like my friend :)

The speedup sounds nice -- I did notice a lag in the popularity load vs. the quotes.

I actually started writing something exactly the same myself to capture this data, but my server died while I was off the grid in India and the scraper lost 1.5 months of time of data lol So coming across your website was awesome
